components/App.js

```diff
@@ -0,0 +1,70 @@
+var prefix = "https://cors-anywhere.herokuapp.com/";
+var GIPHY_API_URL = 'https://api.giphy.com';
+var GIPHY_PUB_KEY = 'EXWUiPgclsIBQDCZxdoceNfovlyag8xm'
+
+App = React.createClass({
+  getInitialState() {
+    return {
+      loading: false,
+      searchingText: '',
+      gif: {}
+    };
+  },
+
+  handleSearch: function(searchingText) {
+    this.setState({
+      loading: true
+    });
+    this.getGif(searchingText, function(gif) {
+      this.setState({
+        loading: false,
+        gif: gif,
```

Comment:
when key and value are same, you can write just one `gif` instead of `gif: gif,`